e2e-tests/e2e-tests.ts

```diff
@@ -88,7 +88,7 @@ async function verifyJourney (matchUser?: boolean) {
 async function eidasEnabledJourney (selectEidas?: boolean, unsignedAssertions?: boolean) {
   await page.goto(host, { waitUntil: 'networkidle2' })
   await page.click('a.button-start')
-  await page.waitForSelector('h1.heading-large')
+  await page.waitForXPath('//h1[text()="Prove your identity to continue"]')
```

Comment:
I much prefer this - it'll be very clear as soon as we make a text change why this breaks.